examples/PubSub/DeclarativeSubscriber/Controllers/Order.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using PubSub.Domain;
+
+namespace DeclarativeSubscriber.Controllers
+{
+    [ApiController]
+    [Route("Order")]
+    public class Order : ControllerBase
+    {
+        private readonly string reporter = "====================\r\n" +
+                                           "OrderId:\t\t{0}\r\n" +
+                                           "OldValue:\t\t{1}\r\n" +
+                                           "NewValue:\t\t{2}\r\n" +
+                                           "OrderUpdateTime:\t{3}\r\n" +
+                                           "====================";
+        [HttpPost]
+        public async Task<OkResult> OnOrderUpdateAsync(PubSubEto<DemoOrderETO> meta)
+        {
+            return await Task.Run(() =>
+            {
+                Console.WriteLine(string.Format(reporter,meta.data.OrderId,meta.data.OldValue,meta.data.NewValue,meta.data.DateTime));
+                
+                //to tell Dapr message ACK.
+                return Ok();
+            });
+        }   
```

Comment:
I'd recommend returning an `ActionResult` or `IActionResult` as it's more generic (allows for failure status codes) and it keeps in line with other examples.

This feels a little forced as a Task/async function. Did you have a specific reason for making it this way?

Reply:
nope, no specific reason.

It's ready to use await in the future.And It can be a general func.